### PR TITLE
Build uWSGI as Python extension class.

### PR DESCRIPTION
**Pull request by jxtx**

### CODE REVIEW

**Comment by martenson** [comment]
@galaxybot build this

**Comment by martenson** [comment]
@galaxybot test this

**Comment by martenson** [comment]
@galaxybot test this

**Comment by martenson** [comment]
@galaxybot test this

File: starforge/forge/wheels.py [hunk 1]
```diff
@@ -218,6 +219,7 @@ def bdist_wheel(self, output=None, uid=-1, gid=-1):
             cmd = [py, 'setup.py'] + build_args
             if platform is not None and self.image.force_plat:
                 cmd.append('--plat-name=%s' % platform)
+            print >> sys.stderr, "!!", cmd
```
Comment (nsoranzo) [inline starforge/forge/wheels.py:222]: Do you want to keep this debugging statement?

**Comment by natefoo** [comment]
This should work with the manylinux images now?

@galaxybot test this

**Comment by jxtx** [comment]
It should work. I tested locally with a locally built container. 

Would appreciate opinions on changing the name of the generated wheel to be different from upstream.

**Comment by natefoo** [comment]
@galaxybot test this

**Comment by natefoo** [comment]
@jxtx see jxtx#1

**Comment by bgruening** [comment]
@jxtx can you rebase this one? Would be great to get this in and get some testing :)

**Comment by jxtx** [comment]
@bgruening I think I clobbered it. Not rebasing well...

**Comment by bgruening** [comment]
I guess in this case a new PR would be the correct way :)
Thanks James!

**Comment by natefoo** [comment]
Superceded by #118.

**Pull request closed without merging**